pwa/src/templates/componentDetail/ComponentsDetailTemplate.module.css

```diff
@@ -45,8 +45,16 @@
   flex: 2;
 }
 
+.logoContainer {
+  max-height: var(--web-app-component-detail-logo-height);
```

Comment:
(See other comment about how we can refactor this)

Note: when setting `max-height`, I'd expect the design token to reflect that (currently it's reflecting `height` and not `max-height`).